### ISSUE TITLE
Design Kitchen

### ISSUE THREAD

**Issue by housemacgill** [houseproject/addition#3]
Initial meeting Tuesday 14th

**Comment by jrmacgill** [comment]
Met with kitchen designer - we have a good initial layout - it does change the size/position of one of the windows in the kitchen.

Designer needs confirmation of room dimension.

**Comment by cutabovenj** [comment]
I am removing the tall double hung window in corner and making the sink window 5'1x3'5 now. I will send new pricing.

**Comment by jrmacgill** [comment]
When do we need to finalize the kitchen design by so as not to delay the project?

Also can you confirm kitchen room dimensions (little hard to pull all the wall lengths from what is labeled on the plans)

**Comment by cutabovenj** [comment]
We have a few weeks on the kitchen design. All of the masonry, framing, roofing, siding, rough electric, plumbing, hvac needs to be done and inspected, framing inspected, insulation install and inspection, drywall, flooring, paint then kitchen install.
I can give you measurements when i get back next week and after Bart has the demo done. I will call it in to Kathy and they have a team come and measure also.

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
Now that demo is mostly done and the position of sewer pipe is a known new obstetrical is now a good time to get the measurements done?

**Comment by cutabovenj** [comment]
Yes, I was going to call Kathy Wednesday to schedule her measure team to come out. Framing will be completed Thursday. My framer and plumber need to be there at the same time to coordinate cutting the existing tub drain, installing a beam and re connecting tub drain all at the same time.

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
Thanks - I see a kitchen design has been marked out on the floor and that electrics / plumbing have gone in based on that.  Is the design based on the preliminary one we did with Kathy and are some decisions now locked in?

**Comment by cutabovenj** [comment]
Well we plan on finishing most plumbing and electrical work thursday. 
If theres any changes you want to make please let me know asap and I can have it re worked Thursday. The mark out in the kitchen is for the new window size and based on the kitchen designs measurements.

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
Sink / Hob / Oven / FF probably in right locations - will check again tomorrow.  Main open question we had was orientation of the island will work that out asap.

**Comment by jrmacgill** [comment]
The hob oven ff may spread out a little as we have more space than thought and we may go for larger options on the hob / oven.  Will know for sure when we meet with Kathy (waiting on a call back)

For the kitchen sink - we can work the kitchen around you - just have the window and sink centered on the wall and let Kathy know where it ends up.

**Comment by cutabovenj** [comment]
You want the window moved to be centered on the back wall now?

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
I want the sink centered on the window wherever the window ends up.  The
window position in the kitchen plan was based on a guess on the final room
width as we did not know it.   What matters most is that we have a single
window.  A sink centered on the window. And the window in a sensible
position on that wall given the width of the room.   We will layout that
wall of the kitchen based on that.

**Comment by cutabovenj** [comment]
This is the larger window for the sink cabinet. I am following the dimensions on Kathys kitchen plan for the center line. 

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
Which may not work because the width of Kathy a plan is wrong

**Comment by jrmacgill** [comment]
That kitchen plan was very rough because we did not know the dimensions of
the room.  I was surprised that she had shared it with you and that it was
taken as final.

**Comment by cutabovenj** [comment]
That is a custom ordered window for the kitchen. We have to reframe the opening so we can put it anywhere on the back wall. Lets get the kitchen design completed before I have the framers move the window opening.

Thank you,
Blaine Shofner
Cut Above Renovations LLC
201-212-0578
www.cutabovenj.com

**Comment by jrmacgill** [comment]
Yes, window size and style not in question.  Now that Kathy has actual room
dimensions we should be able to nail down the layout and position of
window.  Will call her again to try and get an appointment.

**Comment by cutabovenj** [comment]
Ok great. No worries about where to put window. We can move it anywhere on that back wall when we reframe the size. Easy fix/move. I'll shoot her an email as well. I know she wanted the measure guy to get out asap so she can finalize with you.

**Comment by jrmacgill** [comment]
Kitchen plan mostly set - we just need to confirm color for doors and the depth of refrigerator but layout is otherwise set.

Wall oven: Bosch 800 HBL8651UC 
Cooktop : Bosch 800 NGM8655UC

**Comment by jrmacgill** [comment]
We might want to modify the overhang on the island to extend in a second direction.
